sections/rich-text.liquid

```diff
@@ -25,7 +25,7 @@
         {%- for block in section.blocks -%}
           {%- case block.type -%}
             {%- when 'heading' -%}
-              <h2 class="rich-text__heading rte {{ block.settings.heading_size }}" {{ block.shopify_attributes }}>
+              <h2 class="rich-text__heading rte inline-richtext {{ block.settings.heading_size }}" {{ block.shopify_attributes }}>
```

Comment:
I think we can go ahead and make this heading an `inline_richtext` instead or `richtext` here. 
We should be able to mitigate some potential loss of text for merchants: https://github.com/Shopify/shopify/pull/395536

And we should be good to go. We can loop in another dev to double check if we missed anything as well before merging.

Reply:
Since I believe this PR is all set aside from this single change, my inclination is that we should merge this, and then follow up with that change in a simple followup PR once https://github.com/Shopify/shopify/pull/395536 lands.

Reply:
Sounds good to me @kjellr 👍 Ill try and see if I can get a second dev to 👀

Reply:
Yeah let's wait for a second dev approval. I brought it up to @andrewetchen and he should be able to review. 
There were some questions about the performance impact in the editor apparently which we might able to confirm or deny with @stufreen though I'm not sure it would be a blocker anyway.